### DISCUSSION TITLE
Reading discussions without logging in

### DISCUSSION THREAD

**Discussion by adingbatponder** [meshtastic/Discussions#31] [Off Topic]
harassment from moderator

**Comment by barryhunter** [comment]
I can open 
https://github.com/orgs/meshtastic/discussions
in a new broswer without logging in. Also open threads fine. 

Just to login to reply! Screenshot... 

![image](https://github.com/user-attachments/assets/20634771-7f12-41d7-a433-53c9e0fcc0fb)
